### PR TITLE
fix some typos from module contrib to man

### PR DESCRIPTION
**Pull request by lkml-likexu**
Signed-off-by: Aaron.L.Xu <likexu@harmonycloud.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: man/src/container/commit.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ in interactive mode with the bash shell. Apache is also running. To
 create a new image run `docker ps` to find the container's ID and then run:
 
     $ docker commit -m="Added Apache to Fedora base image" \
-      -a="A D Ministrator" 98bd7fc99854 fedora/fedora_httpd:20
```
Comment (vdemeester) [inline man/src/container/commit.md]: This one might be intentional ?

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by cpuguy83** [comment]
Can you squash and provide an accurate commit message?

Thanks!

**Comment by AkihiroSuda** [comment]
@liker12134 
Just a confirmation: are you @xulike666 ?

(I think it is OK to use multiple GitHub accounts, but just confused)

**Comment by xulike666** [comment]
@AkihiroSuda, sorry for the confusion and it‘s my personal private account. 😄

**Review by justincormack — APPROVED** [review]

**Review by cpuguy83 — CHANGES_REQUESTED** [review]
Still needs a commit message update.
Thanks!

**Comment by AkihiroSuda** [comment]
ping @xulike666 (@liker12134)

**Review by thaJeztah — APPROVED** [review]
looks like the commit message is ok now

LGTM

**Pull request merged**